### ISSUE TITLE
Nerfstudio Segmentation fault

### ISSUE THREAD

**Issue by Moreland-cas** [hturki/pynerf#7]
Hallo! Thanks for your great effort!
I came across some environmental issue when I tried to run this repo, here is how I set the environment:

1) first install nerfstudio follow the official steps
pip install torch==2.1.2+cu118 torchvision==0.16.2+cu118 --extra-index-url https://download.pytorch.org/whl/cu118
conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit

2) then install other dependencies
pip install ninja git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
pip install torch-scatter -f https://data.pyg.org/whl/torch-2.1.2+cu118.html   

3) install nerfstudio==0.3.4
and so far ns-train -h works fine 

4) when I tried to install this repo from source pip install -e .
There is a segmentation fault
[pynerf][main] 7s >> ns-train -h
[1]    1908607 segmentation fault (core dumped)  ns-train -h

Any suggestions will be a great help!

**Comment by lethenam** [comment]
I have the same problem. Do you have any solution?
